tests/testthat/test-score_forecasts.R

```diff
@@ -2456,3 +2456,403 @@ test_that("wis is sum of components, 2 intervals, median and point, use_median_a
   
   expect_equal(actual$wis, expected)
 })
+
+test_that("interval coverage does not exist, point forecast only, use_median_as_point FALSE",{
+  y <- c(1, -1, 5)
+  forecast_quantiles_matrix <- rbind(
+    c(-1, 0, 1, 2, 3),
+    c(-2, -1, 2, 2, 4),
+    c(-2, 0, 3, 4, 6))
+  target_end_dates <- lubridate::ymd(20200101) + c(7, 14, 7)
+  horizons <- c("1", "2", "1")
+  locations <- c("01", "01", "02")
+  target_variables <- rep("inc death", length(y))
+  forecast_target_end_dates <-
+    rep(target_end_dates, times = 1)
+  forecast_horizons <- rep(horizons, times = 1)
+  forecast_locations <- rep(locations, times = 1)
+  forecast_target_variables <-
+    rep(target_variables, times = 1)
+  
+  point_forecast <- c(5, 6, 7)
+  
+  test_truth <- data.frame(
+    model = rep("truth_source", length(y)),
+    target_variable = target_variables,
+    target_end_date = target_end_dates,
+    location = locations,
+    value = y,
+    stringsAsFactors = FALSE
+  )
+  
+  n_forecasts <- length(point_forecast)
+  test_forecasts <- data.frame(
+    model = rep("m1", n_forecasts),
+    forecast_date = rep(lubridate::ymd("20200101"), n_forecasts),
+    location = forecast_locations,
+    horizon = forecast_horizons,
+    temporal_resolution = rep("wk", n_forecasts),
+    target_variable = forecast_target_variables,
+    target_end_date = forecast_target_end_dates,
+    type = rep("point", 3),
+    quantile = NA,
+    value = point_forecast,
+    stringsAsFactors = FALSE
+  )
+  actual <- score_forecasts(forecasts = test_forecasts, truth = test_truth, use_median_as_point = FALSE)
+  
+  expect_equal("coverage_0" %in% names(actual),FALSE)
+  
+})
+
+test_that("interval coverage is correct, median only, use_median_as_point TRUE", {
```

Comment:
I think if there is only a median, no interval coverage should be returned

Reply:
There is a coverage 0 essentially saying if they accurately forecasted